app/migrations/Version20170116215417.php

```diff
@@ -20,7 +20,7 @@ class Version20170116215417 extends AbstractMigration
 {
     public function up(Schema $schema): void
     {
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->abortIf(!is_a($this->connection->getDatabasePlatform(), \Doctrine\DBAL\Platforms\MySqlPlatform::class, true), 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
`is_a()` is good method, but for this check it's overkill, use simpler:
```suggestion
        $this->abortIf($this->connection->getDatabasePlatform() instanceof \Doctrine\DBAL\Platforms\MySqlPlatform::class, 'Migration can only be executed safely on \'mysql\'.');
```
Also running cs-fixer to import in `use` that class would be a good idea.

Reply:
In fact, this is still broken on the newer version cause this class exists `<3.0`, in `3.0+` its `MySQLPlatform`

Reply:
While providing the first version I've changed migrations in both places `app/migrations` and `CoreBundle`. While working on "v2" I've omitted accidentally `app/migrations`.